### ISSUE TITLE
compilation error with attribute access

### ISSUE THREAD

**Issue by loveencounterflow** [jashkenas/coffeescript#3560]
i'm feeling a little bit stupid here but i really tried hard to make sure i'm not reporting a ghost. thing is, when i write 

```
d[ 'a' ][ 'b' ][ 'c' ]
```
i get (using coffeescript 1.7.1 and `coffee -c foo.coffee`)
```
error: unexpected 'c'
d[ 'a' ][ 'b' ][ 'c' ]
                 ^^^
```
when i do the same on http://www.compileonline.com/try_coffeescript_online.php i get
```
main.coffee:2:18: error: unexpected STRING
d[ 'a' ][ 'b' ][ 'c' ]
                 ^^^
```
even worse,
```
x = d[ 'a' ][ 'b' ]
x[ 'c' ]
```
gives the same error. what's going on here?

**Comment by lydell** [comment]
Fixed by #3774.

**Comment by loveencounterflow** [comment]
@Delapouite definitely helps, i found out using syntax hiliting in the editor.
